packages/markdown-transform/lib/transform.js

```diff
@@ -221,6 +226,14 @@ const transformationGraph = {
             return t.toCiceroMark(input, options);
         },
     },
+    ooxml: {
```

Comment:
Don't you also need to register the ciceromark_parsed -> ooxml transform?

Reply:
The transformer is converting  both `ooxml<->Ciceromark `

Reply:
My point is it looks like you only registered the ciceromark_parsed -> ooxml transformation. You will need to add a similar block to do the reverse, under a new `ooxml` block.

Reply:
Please check that you can transform from a markdown file to ooxml and the reverse by running the CLI.

Reply:
@dselman 
AFAIR, I was able to convert both transformations `ooxml<->ciceromark_parsed` when I pushed the commit. However, I will recheck and let you know ASAP.

Reply:
@dselman 
Apologies for the delay. I have checked and it converts `CiceroMark_parsed <-> OOXML`. So, this feature is as we expected.